tests/specs/benchmarks/functional-spec.k

```diff
@@ -93,4 +93,6 @@ module FUNCTIONAL-SPEC
        andBool #rangeBytes(32, SIGS)
        andBool notBool #ecrecEmpty(#bufStrict(32, HASH), #bufStrict(32, SIGV), #bufStrict(32, SIGR), #bufStrict(32, SIGS))
 
+    claim <k> runLemma(#range( b"", 0, 1)) => doneLemma(b"\x00") ... </k>
```

Comment:
This syntax is not recognised by the Java backend, and the CI fails. @ehildenb do you know how to get around that, other than commenting this out?

Reply:
I changed this to generate the Memory object in an implementation-agnostic way using `#padLeftBytes`.